packages/create-block/lib/init-package-json.js

```diff
@@ -10,6 +10,29 @@ const writePkg = require( 'write-pkg' );
  */
 const { info, error } = require( './log' );
 
+/**
+ * Resolves the latest version of a package from npm registry
+ *
+ * @param {string} packageName The package name to resolve
+ * @return {Promise<string>} The resolved version range (e.g., "^30.22.0")
+ */
+async function resolvePackageVersion( packageName ) {
+	try {
+		const { stdout } = await command( 'npm', [
+			'view',
+			packageName,
+			'version',
+		] );
```

Comment:
In my testing, the version doesn't get correctly resolved. The following change worked for me:


```suggestion
		const { stdout } = await command( `npm view ${ packageName } version` );
```